packages/rancher-external-ip-webhook/rancher-external-ip-webhook.patch

```diff
@@ -1,40 +1,28 @@
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-external-ip-webhook/charts-original/Chart.yaml packages/rancher-external-ip-webhook/charts/Chart.yaml
---- packages/rancher-external-ip-webhook/charts-original/Chart.yaml	2020-12-11 00:01:16.000000000 +0100
-+++ packages/rancher-external-ip-webhook/charts/Chart.yaml	2020-12-11 00:22:34.000000000 +0100
-@@ -2,12 +2,12 @@
-   catalog.cattle.io/certified: rancher
+--- packages/rancher-external-ip-webhook/charts-original/Chart.yaml	2020-12-11 23:52:10.000000000 +0100
++++ packages/rancher-external-ip-webhook/charts/Chart.yaml	2020-12-11 23:54:42.000000000 +0100
+@@ -3,12 +3,12 @@
    catalog.cattle.io/display-name: External IP Webhook
    catalog.cattle.io/namespace: cattle-externalip-system
+   catalog.cattle.io/os: linux
 -  catalog.cattle.io/release-name: rancher-externalip-webhook
 -  catalog.cattle.io/ui-component: rancher-externalip-webhook
 +  catalog.cattle.io/release-name: rancher-external-ip-webhook
 +  catalog.cattle.io/ui-component: rancher-external-ip-webhook
  apiVersion: v1
- appVersion: v0.1.5
+ appVersion: v0.1.6
```

Comment:
I don't see why this is necessary if we are rebasing to 0.1.6  in the package.yaml

Reply:
We don't set the appVersion??